deploy/template.yaml

```
@@ -262,7 +262,7 @@ Mappings:
       dynatraceSecretArn: arn:aws:secretsmanager:eu-west-2:216552277552:secret:DynatraceNonProductionVariables
       UNIVERSALANALYTICSGTMCONTAINERID: "TBD"
       GOOGLEANALYTICS4GTMCONTAINERID: "TBD"
-      GA4DISABLED: "true"
+      GA4DISABLED: "false"
```

```suggestion
      GA4DISABLED: "true"
```
[We only need to have](https://govukverify.atlassian.net/wiki/spaces/DID/pages/4250730497/GA+4+Package+V1.0.3+Go+No+Go+-+Home#Go-live-steps) GA4 on in staging (where it's already on) and production, so we don't need to make the change in integration

---

Ah ok, now removed the change to integration.